docs/running_in_particular_vpc_subnet.md

```diff
@@ -0,0 +1,47 @@
+# Running jobs in a particular VPC & Subnetwork
+
+To run in a particular VPC and subnetwork you have to make 3 changes. Convert the vcf_to_bigquery.yaml file to the v2alpha1 api, pass the network and subnetwork to the vcf_to_bq command, and set the network, and subnetwork on the virtual machine.
+
+See the example vcf_to_bigquery.yaml below:
```

Comment:
nit: please escape vcf_to_bigquery.yaml with single backquotes (we do this as a convention to separate out parts that are 'code' or 'file names' vs those that are english :) ).